Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -2647,3 +2647,11 @@
     version: "6.0"
     engine: WebKit
     engine_version: "537.36"
+-
+  user_agent: Mozilla/5.0 (Linux x86_64) EkiohFlow/5.11.1.33219 Flow/5.11.1 (like Gecko Firefox/62.0 rv:62.0)
+  client:
+    type: browser
+    name: Flow
+    version: 5.11.1.33219
+    engine: Flow
+    engine_version: 5.11.1.33219
```

Comment:
That actually is the verion of the browser not the engine. Guess we need to change the version parser a bit. It currently uses the engine name to build the regex. Instead it maybe should use the engine regex or we include a special case for flow there...

Reply:
@sgiehl such a result?
```yml
  user_agent: Mozilla/5.0 (Linux x86_64) EkiohFlow/5.11.1.33219 Flow/5.11.1 (like Gecko Firefox/62.0 rv:62.0)
  client:
    type: browser
    name: Flow
    version: 5.11.1
    engine: EkiohFlow
    engine_version: 5.11.1.33219
```

Reply:
From what I've read Flow is the engine, and the version is 5.11.1

Reply:
Yep. that is what I was trying to point out

Reply:
@liviuconcioiu need add detect
Mozilla/5.0 (Linux) AppleWebKit/534.51 (KHTML, like Gecko) Ekioh/2.2.4.7-moto-mob Safari/534.51
Mozilla/5.0 (Linux) AppleWebKit/534.51 (KHTML, like Gecko) Ekioh/2.2.4.7-moto-mob Safari/534.51:534 Motorola KreaTV STB VIP1003
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_0) EkiohFlow/5.7.4.30559 Flow/5.7.4 (like Gecko Firefox/53.0 rv:53.0)

@sgiehl engine with version OK, Engine version = browser version
EkiohFlow string version browser nightly build

I requested information by email and hope to get a response soon. (to be completely sure)

Reply:
I received an official response today.

>  Hi, I'm trying to determine the version of the engine for statistics, for personal purposes. help me figure out what is the browser version and what is the engine string in the useragent.
 useragent `Mozilla/5.0 (Linux x86_64) EkiohFlow/5.11.1.33219 Flow/5.11.1 (like Gecko Firefox/62.0 rv:62.0)`
 
>  EkiohFlow/5.11.1.33219 - engine ?
> Flow/5.11.1 - version browser ?

> Thanks for the answer.

Hi,

We consider the browser engine to be called EkiohFlow, and the browser Flow. Currently they’ll both have the same version (5.11.1, in your example) as there’s only one browser using our browser engine. The ‘33219’ is the repository revision, but has no real purpose for release builds, only debug. The Firefox version number (62) exists simply for compatibility (YouTube), and should be ignored.

I updated the WhichBrowser PHP parser to parse our user agent string, which may be of help. It includes three tests:
https://github.com/WhichBrowser/Parser-PHP/pull/487/files

---- 

Based on the new information, we need to change the name of the browser engine in the definition